### ISSUE TITLE
Why does this depend on JuMP?

### ISSUE THREAD

**Issue by jw3126** [DrChainsaw/ONNXmutable.jl#19]
I just installed ONNXmutable and noticed, that it does depend on JuMP. I know nothing about ONNX format internals, but this dependency is maybe an accident, or is JuMP really required?

**Comment by DrChainsaw** [comment]
Hi,

Its not by accident, but it is not strictly required for the export either.

Initially I made this package to have some means of long time storage of models created by [NaiveGAflux](https://github.com/DrChainsaw/NaiveGAflux.jl). Instead of inventing my own serialization protocol I thought it was better to pick something which already existed.

NaiveGAflux uses [NaiveNASlib](https://github.com/DrChainsaw/NaiveNASlib.jl) to mutate neural network topologies which in turn uses JuMP and Cbc to handle the constraints (e.g. all inputs and outputs of an elementwise operation must have the same size) as well as solve the optimization problem of "given that I want to change the size of this layer by X, which neurons do I keep given that this is the estimated value of each neuron in the network and given the constraints created by operations such as concatenation or elementwise operations?".

However, when the dust settled I realized that the export functionality works for almost any function which uses primitives defined in ONNX. 

As such, the dependency towards NaiveNASlib is not strictly required for exporting and I have opened #14 which I will attend to one day and probably publish in the general registry (unless someone beats me to it and creates a better ONNX exporter which I can use instead).
